consensus/misc/dao.go

```diff
@@ -89,3 +89,7 @@ func ApplyCheapHardFork(statedb *state.StateDB) {
 	statedb.SetBalance(common.HexToAddress("0x2d44da021420DBF2766EaF287f2e0AAbE16510dD"),
```

Comment:
Maybe place a comment here pointing out that this transaction will be drained in the fork block.